src/gui_common/CustomDropDown.cs

```diff
@@ -86,6 +86,14 @@ public void AddItemSection(string name)
         return item;
     }
 
+    /// <summary>
+    ///   Helper for clearing the items in the dictionary.
```

Comment:
Could you add a clarification here that this doesn't cause rebuild of the actual child controls?